test/tests/ver-1gc/comparison_ver-1gc.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+from scipy import special
+import os
+import math
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
+
+#===============================================================================
+# Extract TMAP8 results
+tmap8_sol = pd.read_csv("./gold/ver-1gc_out.csv")
+tmap8_sol_time = tmap8_sol['time']
+tmap8_sol_concentration_A = tmap8_sol['concentration_A']
+tmap8_sol_concentration_B = tmap8_sol['concentration_B']
+tmap8_sol_concentration_C = tmap8_sol['concentration_C']
+
+# conversion from atoms/microns^3 to atoms/m^3
+tmap8_sol_concentration_A = [x*1e18 for x in tmap8_sol_concentration_A]
+tmap8_sol_concentration_B = [x*1e18 for x in tmap8_sol_concentration_B]
+tmap8_sol_concentration_C = [x*1e18 for x in tmap8_sol_concentration_C]
```

Comment:
```suggestion
tmap8_sol_concentration_A *= 1e18
tmap8_sol_concentration_B *= 1e18
tmap8_sol_concentration_C *= 1e18
```
Pythonic way. Also, if we want the values in at/m^3, why do we convert them into at/um^3 in the input file first then back convert to at/m^3? This seems convoluted.

Reply:
Because it helps the convergence a lot to have the values in at/um^3 in the input file.